tools/cli/templates/projects/vite/vite.config.js

```diff
@@ -0,0 +1,6 @@
+export default {
+    // config options
+    build: {
+        cssCodeSplit: false,

```

Comment:
You can also add `assetsInlineLimit: 0` to the config as well, as there is a default limit and if the asset is bellow that it will base64 encode it, which won't work in COHTML.